docs/tutorials/fundamentals/getting_started.md

```diff
@@ -37,7 +37,6 @@ or by dragging and dropping images directly on the viewer.
 We currently only support files that can be read with [`skimage.io.imread`](https://scikit-image.org/docs/dev/api/skimage.io.html#skimage.io.imread),
 such as `tif`, `png`, and `jpg`.
 We plan on adding support for more exotic file types shortly - see [issue #379](https://github.com/napari/napari/issues/379) for discussion.
-You can also create new empty `points`, `shapes`, and `labels` layers using the new layer buttons in the bottom right of the viewer.
```

Comment:
Opinionated change but I think it's documented better in https://napari.org/dev/tutorials/fundamentals/viewer.html#layout-of-the-viewer and it's awkward to describe in text location of these buttons.

Reply:
I like it!

Reply:
I agree and in the same vein, i'd remove the points bit from the example below. I made a comment (I think).